newIDE/app/src/AssetStore/index.js

```diff
@@ -182,77 +179,34 @@ export const AssetStore = ({
     ]
   );
 
-  const resetToDefault = () => {
-    setSearchText('');
-    filtersState.setChosenCategory(null);
-    setOpenedAssetPack(null);
-    setIsAssetPackAdded(false);
-    setOpenedAssetShortHeader(null);
-    clearAllFilters(assetFiltersState);
-    setIsFiltersPanelOpen(false);
-    setIsOnHomePage(true);
-  };
-
   // When a pack is selected from the home page,
   // we set it as the chosen category and open the filters panel.
   const selectPack = (tag: string) => {
     if (!assetPacks) return;
 
     sendAssetPackOpened(tag);
 
-    const assetPack = assetPacks.starterPacks.find(
-      assetPack => assetPack.tag === tag
-    );
-    if (assetPack && assetPack.externalWebLink) {
-      Window.openExternalURL(assetPack.externalWebLink);
-      return;
```

Comment:
I removed this, is it okay?
Was it to open the website of the author?

Reply:
not ok! @4ian 

It just got added to test External Paid Packs :)
You need to keep it

Reply:
How should it works? I made a change to stays on the discovery page when the user click on it because I guess it only redirect to a website. The asset can't be downloaded or browsed from the app store in this version, right?